src/blueprints/tv2/action-factories/tv2-audio-action-factory.ts

```diff
@@ -56,7 +56,7 @@ export class Tv2AudioActionFactory {
       id: 'microphoneUpPiece',
       name: 'Microphone Up',
```

Comment:
Very minor thing, but the ID and names refers to a singular microphone and the methods used, e.g. `createStudioMicrophonesUpTimelineObject` refer to plural microphones. Is there an idea behind that?

Reply:
This is most like from when I did the change from singular to plural I only did it for the TimelineObjectFactory and not the ActionFactory.
I have changed it to plural. I have also changed it from `microphone` to `studioMicrophone`.

Nicely spotted :)